docs/examples/import-definitions/README.md

```diff
@@ -0,0 +1,19 @@
+# Import Definitions Example
+
+You can import definitions, which contains definitions of all broker objects from a local file at Node Boot. (Learn more about export and import rabbitmq definitions)[https://www.rabbitmq.com/definitions.html#import].
+
+You can save the export definitions json file in a ConfigMap object in like so
```

Comment:
'in a ConfigMap object like so'

Reply:
actually, the Google technical writing guide would probably suggest removing 'like so' altogether

"You can save the export definitions json file in a ConfigMap object:"